pkgs/tools/misc/viddy/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "viddy";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "sachaos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1jn5p6rhvizz77qj24j1z241ifn3nz86zh9f3v7cv48z6z9n32vj";
+  };
+
+  vendorSha256 = "b728d1f05adb0ef372fdc487bf59ed208dc70c329f7fd9ee1e7fddd5e3d4dc06";
```

see https://github.com/sachaos/viddy/blob/v0.3.1/.goreleaser.yml#L17-L18
```suggestion
  vendorSha256 = "b728d1f05adb0ef372fdc487bf59ed208dc70c329f7fd9ee1e7fddd5e3d4dc06";
  
  ldflags = [
    "-s" "-w"
    "-X" "main.version=${version}"
  ];
```